app/views/content/steps-to-become-a-teacher.md

```diff
@@ -112,6 +112,8 @@
 
 <b>These steps can help you to achieve qualified teacher status (QTS), which you need to teach in many primary and secondary schools in England.</b>
 
+<b>With QTS you also get additional training and support in your first 2 years as a teacher, helping you to grow in confidence and develop the skills you need to succeed.</b> 
+
```

Comment:
@gemmadallmandfe do you think it may be worth adding in a link to the QTS page here?

Reply:
I don't think it would be right to add it hear as we talk about QTS in other earlier sections on the page.

However, it might be worth adding a link in the intro section - but I'm happy to take your call on this.

![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/a7b2f6a6-cd11-48f4-afb9-89bb71599c89)

Reply:
@gemmadallmandfe added a link into the first mention of QTS